concepts/observability-strategy/configurable-monitoring/01-groundwork.md

```diff
@@ -48,14 +46,12 @@ This concept proposes how to open up to those new scenarios by making integratio
 - Envoy metrics should be collected instantly (without user action).
 
 ### Local backend
-- Installing the monitoring component ships managed Prometheus instances.
+- Kyma will provide a blueprint based on Helm for installing the kube-prometheus-stack.
 - The setup is not meant to be HA and scalable.
-- The setup should handle overload properly, so that it is always in a functional state.
-- Multiple instances might be used for the different scenarios like system, istio, and custom. That will assure reliability of one scenario if the other is in trouble.
 
-## General Idea
+## Proposed Solution
 
-The idea of the proposal is to introduce a preconfigured agent layer that's responsible for collecting metric data only. Those agents can be configured dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes APIs. The agent and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution that can be installed optionally and will stay in the existing modules like `monitoring`.
+The idea of the proposal is to introduce a preconfigured agent layer that's responsible for collecting metric data only. Those agents can be configured dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes APIs. The agent and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. They can be installed manually by the user following a blueprint.
```

Comment:
```suggestion
The proposal introduces a new preconfigured agent layer that's responsible for collecting metric data only. Users can configure those agents dynamically at runtime with different configuration scenarios, so that the agents start shipping the data to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes APIs. The agent and the new operator are bundled in a new core package called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. They can be installed manually by the user following a blueprint.
```